builder/builder_test.go

```diff
@@ -0,0 +1,44 @@
+package builder
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func mustExist(t *testing.T, name string) {
+	t.Helper()
+	_, err := os.Stat(name)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func cleanup(file string) {
```

Comment:
nitpick - you call this on a directory, not a file (ok directories are still files, but still...)